samples/XAMLCatalog/XAMLCatalog/UIKitControls/UIButtonWithControlsViewController.h

```diff
@@ -0,0 +1,25 @@
+//******************************************************************************
+//
+// Copyright (c) 2015 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import "MenuTableViewController.h"
+
+@interface UIButtonWithControlsViewController : UIViewController <UITextFieldDelegate>
```

Comment:
>UITextFieldDelegate [](start = 66, length = 19)

i don't see you implement any UITextField delegate method, instead, you registered to listen to controChangedlEvent. Are you sure that is what you want?